src/MODULE.tools

```diff
@@ -41,7 +41,8 @@ bazel_dep(name = "rules_java", version = "8.15.2")
 bazel_dep(name = "rules_cc", version = "0.2.8")
 bazel_dep(name = "rules_python", version = "1.4.1")
 bazel_dep(name = "rules_shell", version = "0.6.1")
-bazel_dep(name = "apple_support", version = "1.22.1", repo_name = None)
+bazel_dep(name = "apple_support", version = "1.23.1", repo_name = None)
+bazel_dep(name = "rules_apple", version = "4.0.0", repo_name = None) # version bump, older versions contribute lockfile entries
```

Comment:
Make this `4.0.1` to match the root MODULE.bazel could probably fix the bootstrap test